diff -dupr old/zserik-common-0.1/base.sh new/zserik-common-0.1/base.sh
--- old/zserik-common-0.1/base.sh	2013-11-25 19:11:05.000000000 +0100
+++ new/zserik-common-0.1/base.sh	2013-11-29 21:36:44.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 source /lib/rc/sh/functions.sh
 
diff -dupr old/zserik-common-0.1/color.sh new/zserik-common-0.1/color.sh
--- old/zserik-common-0.1/color.sh	2013-11-25 19:10:59.000000000 +0100
+++ new/zserik-common-0.1/color.sh	2013-11-29 21:38:32.000000000 +0100
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/bin/sh
+
+source /lib/rc/sh/functions.sh
 
 if yesno ${ZS_COLOR:-YES} || yesno ${EINFO_COLOR:-YES} ; then
 	BOLD=$(tput bold)
diff -dupr old/zserik-common-0.1/misc.sh new/zserik-common-0.1/misc.sh
--- old/zserik-common-0.1/misc.sh	2013-11-25 19:09:33.000000000 +0100
+++ new/zserik-common-0.1/misc.sh	2013-11-29 21:36:44.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 source /usr/share/zserik-common/base.sh
 
diff -dupr old/zserik-common-0.1/umlauts.sh new/zserik-common-0.1/umlauts.sh
--- old/zserik-common-0.1/umlauts.sh	2013-11-25 19:07:43.000000000 +0100
+++ new/zserik-common-0.1/umlauts.sh	2013-11-29 21:36:44.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 [ "${AE}" ] || AE=$'\303\244'
 [ "${OE}" ] || OE=$'\303\266'
