diff --git a/CMakeLists.txt b/CMakeLists.txt
index 68f1564..0d79d88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -190,6 +190,9 @@ add_definitions(-DLIZARDFS_HAVE_CONFIG_H)
 add_definitions(-D__STDC_LIMIT_MACROS)
 add_definitions(-D__STDC_CONSTANT_MACROS)
 add_definitions(-D__STDC_FORMAT_MACROS)
+add_definitions(-DSPDLOG_FMT_EXTERNAL)
+add_definitions(-DFMT_HEADER_ONLY)
+
 if(MINGW)
   add_definitions(-DWINVER=0x0602)
   add_definitions(-D_WIN32_WINNT=0x0602)
diff --git a/cmake/Libraries.cmake b/cmake/Libraries.cmake
index 0284f74..67a19de 100644
--- a/cmake/Libraries.cmake
+++ b/cmake/Libraries.cmake
@@ -6,7 +6,7 @@ if(ENABLE_TESTS)
 endif()
 
 # Find spdlog
-
+find_package(fmt REQUIRED)
 find_package(spdlog REQUIRED)
 
 # Find standard libraries
