diff -ru a/actions/main b/actions/main
--- a/actions/main	2016-01-20 19:04:15.000000000 +0100
+++ b/actions/main	2016-01-20 19:07:31.905541938 +0100
@@ -20,7 +20,7 @@
 
 [ -f "$RMLIST" ] && while read FILE; do
     [ -f "$FILE" ] || continue
-    FILE="$(realpath "FILE")"
+    FILE="$(realpath "$FILE")"
     echo RM "$FILE"
     mv -f "$FILE" "$(dirname "$FILE")/.archive/$(basename "$FILE")"
 done < "$RMLIST"
